mimic/model/ironic_objects.py

```diff
@@ -0,0 +1,290 @@
+        Brief JSON-serializable version of this server, for the non-details
+        list nodes request.
+        """
+        return {
+            "instance_uuid": self.instance_uuid,
+            "uuid": self.node_id,
+            "links": self.links_json(),
+            "maintenance": self.maintenance,
+            "provision_state": self.provision_state,
+            "power_state": self.power_state,
+            "name": self.name
+        }
+
+    def detail_json(self):
+        """
+        Long-form JSON-serializable object representation of this node, as
+        returned by either a GET on this individual node or a member in the
+        list returned by the list-details request.
+        """
+        template = self.static_defaults.copy()
+        template.update({
+            "instance_uuid": self.instance_uuid,
+            "name": self.name,
+            "uuid": self.node_id,
+            "links": self.links_json(),
+            "maintenance": self.maintenance,
+            "provision_state": self.provision_state,
+            "power_state": self.power_state,
+            "ports": self.port_links_json(),
+            "extra": {
+                "flavor": self.flavor_id,
+                "hardware/inventory/disks/0/size": "31016853504",
+                "hardware/interfaces/0/switch_port_id": "Mimic0/01",
+                "rackid": "Mimic_rackid",
+                "hardware/interfaces/1/switch_chassis_id": "mimic-chassis1",
+                "core_id": "00000",
+                "hardware/inventory/disks/2/size": "1861817990000",
+                "uutsn": "Mimic-uutsn",
+                "hardware/inventory/disks/1/size": "1861817990000",
+                "hardware/inventory/cpu/count": "40",
+                "hardware/inventory/memory/total": "135234740224",
+                "hardware/inventory/disks/2/rotational": "False",
+                "hardware/inventory/disks/0/rotational": "False",
+                "hardware/inventory/disks/1/rotational": "False",
+                "racklocation": "Mimic00",
+                "hardware/interfaces/1/switch_port_id": "Mimic1/10",
+                "hardware/interfaces/0/switch_chassis_id": "mimic-chassis1"
+            },
+            "properties": {
+                "memory_mb": self.memory_mb,
+                "cpu_arch": "amd64",
+                "local_gb": 32,
+                "cpus": 40
+            },
+            "driver_info": {
+                "hardware_manager_version": None,
+                "ipmi_username": "USERID",
+                "ipmi_address": "127.0.0.0",
+                "decommission_target_state": None,
+                "ipmi_password": "******",
+                "agent_url": "http://127.0.0.1:9999",
+                "agent_last_heartbeat": 1427727371,
+                "cache_image_id": self.cache_image_id,
+                "cache_status": 'cached' if self.cache_image_id else None
+            }
+        })
+        template["properties"]["memory_mb"] = self.memory_mb
```

Comment:
This line may not be needed, since it's already set in the update dictionary above.